Stop talking about Pervasives in core library documentation

nobrowser opened the pull request:
Since `Pervasives` is now just a deprecated alias for `Stdlib`, the Manual should stop mentioning it as a real module.

Commit:
Stop talking about Pervasives in core library documentation

Commit:
Add Changelog entry

dra27 commented on Changes:
```diff
@@ -295,6 +295,9 @@ OCaml 4.09.0
 
 ### Manual and documentation:
 
+- #8757: Stop talking about Pervasives in core library documentation.
```
```suggestion
- #8757: Rename Pervasives to Stdlib in core library documentation.
```

dra27 reviewed (commented):
Thanks - I’d change the description of the change, but the change itself is of course correct.

dra27 committed:
Rename references to Pervasives to Stdlib

dra27 committed:
Choose a better example for external

Pervasives.input has never been a primitive (it was always unsafe_input)
- string_of_int has the benefit of being declared external.

dra27 commented:
I pushed a commit with a few other renames and, while there, spotted that the use of `input` as an example in the C chapter of the manual was a bit limited. This all seems good for 4.09 as well, being documentation fixes only.

ghost commented:
Thanks @dra27 , that scratches my itch thoroughly.

pmetzger commented:
I'm confused, did this get closed without being merged?

ghost commented:
> I'm confused, did this get closed without being merged?

I thought @dra27 said he pushed a commit which included my suggestions plus other changes.  Have I misunderstood him?

dbuenzli commented:
> I thought @dra27 said he pushed a commit which included my suggestions plus other changes. Have I misunderstood him?

Yes he added them on you branch (see above) but he didn't merge the PR.

pmetzger commented:
Gotcha.

dra27 commented:
trunk: https://github.com/ocaml/ocaml/commit/466950b48f698cb2d9fbb5571a3ed203a8765044
4.08: https://github.com/ocaml/ocaml/commit/d44685bf54df51abffef558d92af7f014a88fdd4
4.09: https://github.com/ocaml/ocaml/commit/e9f72e515a540e62657e56f02ec1128ec98f074b
